source/views/sis/balances.tsx

```diff
@@ -13,13 +13,14 @@ import {BalancesShapeType, useBalances} from '../../lib/financials'
 import * as c from '@frogpond/colors'
 import {sto} from '../../lib/colors'
 import {useNavigation} from '@react-navigation/native'
-import {NoCredentialsError, useCredentials} from '../../lib/login'
+import {NoCredentialsError, useUsername} from '../../lib/login'
 
 const DISCLAIMER = 'This data may be outdated or otherwise inaccurate.'
 
 export const BalancesView = (): JSX.Element => {
 	let navigation = useNavigation()
-	let credentials = useCredentials()
+	let {data: usernameData} = useUsername()
+	let username = usernameData ? usernameData.username : ''
```

Comment:
```suggestion
	let username = usernameData?.username ?? ''
```

I could go either way on this one.

Reply:
I did like this better! Can we centralize this in the username hook, though? I get that changes the expectation to return an empty string, though. I'm trying to figure out what a clean implementation could be for the consumers.

Reply:
I think I'd rather make the consumers do this… but idk. It's pretty short…